File: client/src/main/java/com/oneinlet/disruptor/MessageEvent.java
package com.oneinlet.disruptor;

/**
 * Created by WangZiHe on 19-8-15
 * QQ/WeChat:648830605
 * QQ-Group:368512253
 * Blog:www.520code.net
 * Github:https://github.com/yancheng199287
 */

//定义事件数据，用于在事件中传递的数据对象
public class MessageEvent {

    private Integer id;
    private Long number;
    private String name;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getNumber() {
        return number;
    }

    public void setNumber(Long number) {
        this.number = number;
    }

    @Override
    public String toString() {
        return "MessageEvent{" +
                "id=" + id +
                ", number=" + number +
                ", name='" + name + '\'' +
                '}';
    }
}

File: client/src/main/java/com/oneinlet/disruptor/MessageEvent.kt
package com.oneinlet.disruptor

import com.oneinlet.common.bean.FileAction
import java.io.File

/**
 * Created by WangZiHe on 19-8-15
 * QQ/WeChat:648830605
 * QQ-Group:368512253
 * Blog:www.520code.net
 * Github:https://github.com/yancheng199287
 */

data class MessageEvent(
        var sequence: Long? = null,//序号
        var file: File? = null,
        var fileAction: FileAction? = null
)